### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pghrogue**

### CODE REVIEW

File: backend/data/helpers/userModel.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const db = require('../dbConfig.js');
+
+module.exports = {
+  // get - with and without id
+  get: function(id) {
+    let query = db('users');
```
Comment (KingAtoki) [inline backend/data/helpers/userModel.js]: Another way of doing this
```suggestion
    return id ? db('users').where('user_id', id).first() : db('users')
```

File: backend/routes/userRouter.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+const router = express.Router();
+
+// App requires/middleware
+const user = require('../data/helpers/userModel');
+
+
+/* ---------- Endpoints for /api/user ---------- */
+
+/* GET (list) */
+router.get( '/', (req, res) => {
```
Comment (KingAtoki) [inline backend/routes/userRouter.js:12]: Shouldn't this have `id` in the url
```suggestion
router.get( '/:id', (req, res) => {
```

**Review by KingAtoki — APPROVED** [review]
Made a few suggestions if you want to take a look, but otherwise everything looks good!

File: backend/data/helpers/userModel.js (hunk 1 above)
Comment (pghrogue) [reply backend/data/helpers/userModel.js]: I like that! Thanks!

**Pull request merged**